### PR TITLE
Units mnrva 801

### PR DESCRIPTION
**Pull request by FarajiA**
## JIRA:
See ticket here - https://jira.rax.io/browse/MNRVA-801

## Description:
Add support for initial units of measurements. Update documentation related to multi group & units

```
    "percentage": "%",
    "bytes": "b",
    "packets/sec": "packets/s",
    "kilobytes": "kb",
    "megabytes": "mb",
    "kilobytes/second": "kb/s",
    "available": "available"
```
## Testing:
Add one of the values above to the `data-unit` attribute 

## Screenshots:
(if applicable)

### CODE REVIEW

File: docs/Gemfile.lock [hunk 1]
```diff
@@ -4,17 +4,17 @@ GEM
     addressable (2.7.0)
       public_suffix (>= 2.0.2, < 5.0)
     colorator (1.1.0)
-    concurrent-ruby (1.1.6)
-    em-websocket (0.5.1)
+    concurrent-ruby (1.1.9)
```
Comment (lovedeep92) [inline docs/Gemfile.lock:7]: Is these updates are required in gem file?

Comment (FarajiA) [reply docs/Gemfile.lock:7]: I'm not sure. I just ran the `bundle update` command and this is what I got. To be sure it works for you checkout the branch and run the docs. These updates were automatically applied

**Review by pras7430 — APPROVED** [review]
LGTM!

File: src/helpers/supportedInfo.js [hunk 2]
```diff
@@ -10,35 +10,21 @@ import { graphs } from './supported.json';
...
+     * Configure the unit of measurment to be used
      * @param {string} type
-     * @returns {object}
+     * @returns string
      */
-    info(type, field) {
-        let supportedInfo;
-        if (type === "" || type === undefined) {
+    configureUnits(type) {
+        if (!(!!type)) {
```
[the remark this answers is not in the document]
Comment (FarajiA) [reply src/helpers/supportedInfo.js:23]: lol it does once compiled. The parentheses is not needed, is there a more preferred way to make this condition?

**Pull request merged**